### ISSUE TITLE
Calendar with showTime on midnight

### ISSUE THREAD

**Issue by Helayxa** [primefaces/primeng#5800]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
I use a Calendar with showTime. When I change the hour value to set 0, it displays 12 in the input.

**Expected behavior**
When I choose 0 as hour, it should show 0 in the input

**Minimal reproduction of the problem with instructions**
https://stackblitz.com/edit/github-q9fs7c?file=src/app/app.component.html

**Comment by Mysame** [comment]
This is solved in 5.2.7 , plese upgrade your project to this version.

**Comment by cagataycivici** [comment]
I'm unable to replicate with the latest PrimeNG. If the issue still persists with the new version please create a new issue with our stackblitz template.
